src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax;
+
+import core.basesyntax.impl.ActivitiesStrategyImpl;
+import core.basesyntax.service.FileReader;
+import core.basesyntax.service.FileWriter;
+import core.basesyntax.service.ReportGenerator;
+import core.basesyntax.service.TransactionProcess;
+import core.basesyntax.strategy.ActivitiesHandler;
+import core.basesyntax.strategy.ActivitiesStrategy;
+import core.basesyntax.strategy.BalanceActivitiesHandler;
+import core.basesyntax.strategy.PurchaseActivitiesHandler;
+import core.basesyntax.strategy.ReturnActivitiesHandler;
+import core.basesyntax.strategy.SupplyActivitiesHandler;
+import java.util.Arrays;
+import java.util.HashMap;
+
+public class Main {
+    public static void main(String[] args) {
+        final String reportFile = "report.csv";
+        final String path = "src/main/resources/file.csv";
```

Comment:
It should be constant. Also add path to report file 
```suggestion
    private static final String INPUT_FILE = "src/main/resources/input.csv";
    private static final String OUTPUT_FILE = "src/main/resources/report.csv";
```